validation-test/stdlib/XCTest.swift

```diff
@@ -3,11 +3,9 @@
 
 // REQUIRES: objc_interop
 
-// Currently it fails because a dylib cannot be found.
-// TODO: Re-enable this test when rdar://problem/24222804 is fixed
```

Comment:
Please keep this comment.  The issue is not resolved, since the test is still disabled on non-OS X.